_maps/map_files/BoxStation/BoxStation_Skyrat.dmm

```diff
@@ -20476,7 +20488,7 @@
 /area/storage/tools)
 "aTM" = (
 /obj/structure/table,
-/obj/item/reagent_containers/food/snacks/mint,
```

Comment:
Removing the only mint

Reply:
Is the mint really that important?

Reply:
I mean I don't know why it was removed

Reply:
It was removed to move the scrambler thingy that grinds things over

Reply:
Please readd it back somewhere else

Reply:
I mean i find this pretty petty. But sure.